inspectors/view/qml/notation/notes/HeadSettings.qml

```diff
@@ -41,6 +41,27 @@ FocusableItem {
             noteHeadTypesModel: root.model ? root.model.noteheadTypesModel : null
         }
 
+        CheckBox {
+            isIndeterminate: {
+                if (!model)
+                    return false;
+                if (model.isUndefined)
+                    return false;
+
+                return model.parenthesesNotehead.value === NoteHead.PARENTHESIS_RIGHT || model.parenthesesNotehead.value === NoteHead.PARENTHESIS_LEFT
+            }
+            checked: (model && !isIndeterminate) ? (model.parenthesesNotehead.value === NoteHead.PARENTHESIS_ALL) : false
+            text: qsTr("Bracket Notehead")
+
+            onClicked: {
+                if (isIndeterminate) {
+                    model.parenthesesNotehead.value = NoteHead.PARENTHESIS_ALL
```

Comment:
missing `;`

Reply:
Are they needed in qml?

Reply:
Probably not, but it's best to have either semicolons for all statements in a function-like block, or no semicolons.

Reply:
is there an equivalent to javascript `use strict`? Else, I will probably miss many more...